client/my-sites/site-settings/reading-newsletter-settings/SubscribeModalSetting.tsx

```diff
@@ -24,6 +30,20 @@ export const SubscribeModalSetting = ( {
 	disabled,
 }: SubscribeModalSettingProps ) => {
 	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId ) as number;
+
+	// Construct a link to edit the modal
+	const { data: activeThemeData } = useActiveThemeQuery( siteId, true );
+	const isFSEActive = activeThemeData?.[ 0 ]?.is_block_theme ?? false;
+	const themeSlug = activeThemeData?.[ 0 ]?.template;
```

Comment:
This could be `template` or `style` — both have same value, but I'm not sure how differnece affects. For now this works so can can loop back to change if needed.